### PR TITLE
Wcman kalkulashka

### PR DESCRIPTION
**Pull request by Toiletman007**

### CODE REVIEW

**Comment by Toiletman007** [comment]
[Kalkulačka](https://trello.com/c/MiNgrGCW/43-kalkula%C4%8Dka)

File: src/main/java/org/cakelook/Main.java [hunk 1]
```diff
@@ -1,7 +1,44 @@
...
+            case "/":
+                if (num2 != 0){
+                    output = (float) num1 / num2;
+                }else{
+                    System.out.println("no deviding bye 0 or elz !!!! D:<");
+                }
+                break;
+            case "*":
+                output = num1*num2;
+                break;
```
Comment (Kostahe) [inline src/main/java/org/cakelook/Main.java:35]: Enum better option

File: src/main/java/org/cakelook/Main.java [hunk 2]
```diff
@@ -1,7 +1,44 @@
...
+            case "+":
+                output = num1 + num2;
+                break;
+            case "-":
+                output = num1 - num2;
+                break;
+            case "/":
+                if (num2 != 0){
+                    output = (float) num1 / num2;
+                }else{
```
Comment (Kostahe) [inline src/main/java/org/cakelook/Main.java:29]: Uletelo formatovani control + alt + L

File: src/main/java/org/cakelook/Main.java [hunk 3]
```diff
@@ -1,7 +1,44 @@
...
+            case "*":
+                output = num1*num2;
+                break;
 
+            default:
+                System.out.println("please use one of the symbols above");
+                break;
+        }
+        System.out.println(output);
+        scanner.close();
```
Comment (Kostahe) [inline src/main/java/org/cakelook/Main.java:42]: Perfect

**Review by Kostahe — APPROVED** [review]
FORMATOVANI I WON'T APPROVE NEXT TIME 😡

**Pull request merged**